src/aura/HH_AutoCompleteOption/HH_AutoCompleteOption.cmp

```diff
@@ -3,29 +3,14 @@
         <span>
             <div class="slds-media slds-tile slds-hint-parent">
                 <div class="slds-media__figure">
-                    <aura:if isTrue="{!v.value.HHId__c != null}" >
-                        <c:svg ariaHidden="true"
-                               class="slds-icon slds-icon--small slds-icon-standard-household"
-                               xlinkHref="{!$Resource.sfdo203 + '/assets/icons/standard-sprite/svg/symbols.svg#household'}"
-                               />
-                        <aura:set attribute="else">                            
-                            <c:svg ariaHidden="true"
-                                   class="slds-icon slds-icon--small slds-icon-standard-contact"
-                                   xlinkHref="{!$Resource.sfdo203 + '/assets/icons/standard-sprite/svg/symbols.svg#contact'}"
-                                   />
-                        </aura:set>
-                    </aura:if>
+                    <lightning:icon iconName="{!'standard:' + (!empty(v.value.HHId__c) ? 'household' : 'contact')}"
```

Comment:
I will double check this as well, but in my org I'm definitely seeing 2 different icons in my org.